editor/editor_node.cpp

```diff
@@ -1726,7 +1726,9 @@ void EditorNode::_find_node_types(Node *p_node, int &count_2d, int &count_3d) {
 	}
 }
 
+// **Deprecated** now generates preview at EditorPackedScenePreviewPlugin
 void EditorNode::_save_scene_with_preview(String p_file, int p_idx) {
+	return; // Disable
```

Comment:
This method is still indirectly called by `EditorInterface.save_scene_as()` (when the second parameter is `true` which is the default). This is exposed to scripting.

However, can't we remove this particular method and stop calling it in `save_scene_to_path()`?

The `with_preview` parameter of `EditorInterface.save_scene_as()` should then be marked as deprecated in the documentation:

```xml
		<method name="save_scene_as">
			<return type="void" />
			<param index="0" name="path" type="String" />
			<param index="1" name="with_preview" type="bool" default="true" />
			<description>
				Saves the currently active scene as a file at [param path].
				[b]Note:[/b] The [param with_preview] parameter has no effect.
			</description>
		</method>
```